utils/2fa.js

```diff
@@ -0,0 +1,55 @@
+
+const prompts = require('prompts');
+const {
+    multisig: {
+        AccountMultisig: AccountMultisigApi,
+    },
+} = require('near-api-js');
+
+class AccountMultisig extends AccountMultisigApi {
+    constructor(connection, accountId) {
+        super(connection, accountId);
+    }
+    // override for custom UX
```

Comment:
how does it work when we don't override? Does it make sense to allow to provide some callbacks for input when creating `Account2FA`?

Reply:
It will try to signAndSendTransaction without sending code from helper. You must override it and call helper methods, unless you have another way to call confirm on user's account.

Reply:
seems like `Account2FA ` should be able to pass callbacks instead, like:

```
new Account2FA(connection, accountId, {
     getSecurityCode: async function() {
         //  Loop to query user input in console goes here
     }
});
```